homeassistant/components/stream/__init__.py

```diff
@@ -464,3 +471,26 @@ def get_diagnostics(self) -> dict[str, Any]:
 def _should_retry() -> bool:
     """Return true if worker failures should be retried, for disabling during tests."""
     return True
+
+
+STREAM_OPTIONS_SCHEMA: Final = vol.Schema(
+    {
+        vol.Optional(CONF_RTSP_TRANSPORT): vol.In(RTSP_TRANSPORTS),
```

Comment:
Does CONF_USE_WALLCLOCK_AS_TIMESTAMPS need to be in this list now?

Reply:
Yup, thanks